### DISCUSSION TITLE
Data display of „Solar Gain Today“ different to „Pellet Usage Today“

### DISCUSSION THREAD

**Discussion by trudetta** [dominikamann/oekofen-pellematic-compact#45] [Q&A]
Hello Dominik, 

I am really happy about you Oekofen integration. Thanks for the great work.

Recently I got my solar modules installed and now I wanted to update my dashboard with displaying the solar gain per day in a bar chart, as I am doing it for the pellet usage per day. Unfortunately it does not work as the data display seems to be different. Checking the “Verlauf” of both entities, the different behavior is directly visible, can you explain this?
![IMG_3975](https://github.com/dominikamann/oekofen-pellematic-compact/assets/155584372/fb4926f7-5219-466c-906a-25e27a20e1e4)
![IMG_3976](https://github.com/dominikamann/oekofen-pellematic-compact/assets/155584372/7dc1cb3d-262c-45a9-b98d-abcfafbaee68)

**Comment by dominikamann** [comment]
Hi, 

this is beacuse the sensor was added as "string" to HA. I need to know the exact data types.
Can you please tell me the correct units and factors?

It is possible if you open the API in help mode: http://[YOUR_PELLEMATIC_IP]:4321/[YOUR_PASSWORD]/all?

You only need to copy the result and provide it here.

All the best.

**Reply by trudetta** [reply]
Hey Dominik, thanks for your reply. Below the results when opening API:

{
 "system":{
  "system_info":"system global variables",
  "L_ambient":{"val":87, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Außentemperatur"}, 
  "L_errors":{"val":0, "factor":1, "min":-32768, "max":32767, "text":"Fehler"}, 
  "L_usb_stick":{"val":0, "format":"0:Aus|1:Ein", "text":"Usb Stick erkannt"}, 
  "L_existing_boiler":{"val":0, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Bestehender Kessel"}, 
  "mode":{"val":1, "format":"0:Aus|1:Auto|2:Warmwasser", "text":"Betriebsart"}
 },
 "weather":{
  "weather_info":"current weather data",
  "L_temp":{"val":100, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Akt. Temperatur"}, 
  "L_clouds":{"val":0, "unit":"%", "factor":1, "min":-32768, "max":32767, "text":"Akt. Bewölkung"}, 
  "L_forecast_temp":{"val":120, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Durchschnittliche Temperatur"}, 
  "L_forecast_clouds":{"val":79, "unit":"%", "factor":1, "min":-32768, "max":32767, "text":"Durchschnittliche Bewölkung"}, 
  "L_forecast_today":{"val":0, "format":"0:Heute|1:Morgen", "text":"Wetterprognose start"}, 
  "L_starttime":{"val":910, "factor":1, "min":-32768, "max":32767, "text":"Startzeit"}, 
  "L_endtime":{"val":1620, "factor":1, "min":-32768, "max":32767, "text":"Endzeit"}, 
  "L_source":{"val":"https://www.openweathermap.org", "length":20, "text":"Info"}, 
  "L_location":{"val":"Dortmund|Germany|2935517", "length":20, "text":"Ort"}, 
  "cloud_limit":{"val":55, "unit":"%", "factor":1, "min":0, "max":100, "text":"Bewölkungslimit"}, 
  "hysteresys":{"val":-40, "unit":"K", "factor":0.1, "min":-200, "max":0, "text":"Abbruchtmp. Differenz"}, 
  "offtemp":{"val":-100, "unit":"°C", "factor":0.1, "min":-300, "max":200, "text":"Abschalttemperatur"}, 
  "lead":{"val":120, "unit":"min", "factor":1, "min":0, "max":600, "text":"Vorhaltezeit"}, 
  "refresh":{"val":0, "text":"Aktualiseren"}, 
  "oekomode":{"val":0, "format":"0:Aus|1:Ein", "text":"Öko Modus"}
 },
 "forecast":{
  "forecast_info":"date|temp|cloud|speed|image|code|unit[|sunrise|sunset] code see https://openweathermap.org/weather-conditions",
  "L_w_0":{"val":"Fr, 8 Mär 17:53|10|0|22 km/h|01d|800|C|06:59|18:22", "length":20}, 
  "L_w_1":{"val":"Fr, 8 Mär 19:00|10|1|12 km/h|01n|800|C", "length":20}, 
  "L_w_2":{"val":"Fr, 8 Mär 22:00|7|0|12 km/h|01n|800|C", "length":20}, 
  "L_w_3":{"val":"Sa, 9 Mär 01:00|5|0|9 km/h|01n|800|C", "length":20}, 
  "L_w_4":{"val":"Sa, 9 Mär 04:00|4|2|8 km/h|01n|800|C", "length":20}, 
  "L_w_5":{"val":"Sa, 9 Mär 07:00|3|15|8 km/h|02d|801|C", "length":20}, 
  "L_w_6":{"val":"Sa, 9 Mär 10:00|7|91|8 km/h|04d|804|C", "length":20}, 
  "L_w_7":{"val":"Sa, 9 Mär 13:00|12|91|9 km/h|04d|804|C", "length":20}, 
  "L_w_8":{"val":"Sa, 9 Mär 16:00|13|68|11 km/h|04d|803|C", "length":20}, 
  "L_w_9":{"val":"Sa, 9 Mär 19:00|9|45|9 km/h|03n|802|C", "length":20}, 
  "L_w_10":{"val":"Sa, 9 Mär 22:00|8|95|6 km/h|04n|804|C", "length":20}, 
  "L_w_11":{"val":"So, 10 Mär 01:00|7|97|8 km/h|04n|804|C", "length":20}, 
  "L_w_12":{"val":"So, 10 Mär 04:00|5|100|10 km/h|04n|804|C", "length":20}, 
  "L_w_13":{"val":"So, 10 Mär 07:00|4|97|10 km/h|04d|804|C", "length":20}, 
  "L_w_14":{"val":"So, 10 Mär 10:00|8|95|12 km/h|04d|804|C", "length":20}, 
  "L_w_15":{"val":"So, 10 Mär 13:00|12|98|18 km/h|04d|804|C", "length":20}, 
  "L_w_16":{"val":"So, 10 Mär 16:00|13|100|17 km/h|04d|804|C", "length":20}, 
  "L_w_17":{"val":"So, 10 Mär 19:00|10|100|12 km/h|04n|804|C", "length":20}, 
  "L_w_18":{"val":"So, 10 Mär 22:00|8|100|12 km/h|04n|804|C", "length":20}, 
  "L_w_19":{"val":"Mo, 11 Mär 01:00|8|100|8 km/h|04n|804|C", "length":20}, 
  "L_w_20":{"val":"Mo, 11 Mär 04:00|8|100|8 km/h|04n|804|C", "length":20}, 
  "L_w_21":{"val":"Mo, 11 Mär 07:00|7|100|9 km/h|04d|804|C", "length":20}, 
  "L_w_22":{"val":"Mo, 11 Mär 10:00|8|100|7 km/h|04d|804|C", "length":20}, 
  "L_w_23":{"val":"Mo, 11 Mär 13:00|9|100|7 km/h|04d|804|C", "length":20}, 
  "L_w_24":{"val":"Mo, 11 Mär 16:00|9|100|11 km/h|04d|804|C", "length":20}
 },
 "hk1":{
  "hk_info":"heating circuit data",
  "L_roomtemp_act":{"val":0, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Raumtemperatur"}, 
  "L_roomtemp_set":{"val":220, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"RT Soll"}, 
  "L_flowtemp_act":{"val":459, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"VL Ist"}, 
  "L_flowtemp_set":{"val":449, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"VL Soll"}, 
  "L_comfort":{"val":0, "unit":"K", "factor":0.1, "min":-32768, "max":32767, "text":"Komforttemperatur"}, 
  "L_state":{"val":32, "factor":1, "text":"Status"}, 
  "L_statetext":"Heizbetrieb aktiv",
  "L_pump":{"val":1, "format":"0:Aus|1:Ein", "text":"HK Pumpe"}, 
  "remote_override":{"val":0, "unit":"K", "factor":0.1, "min":-32768, "max":32767, "text":"HK Fernbedienung"}, 
  "mode_off":{"val":0, "format":"0:Aus|1:Auto|2:Heizen|3:Absenken", "text":"Betriebsart"}, 
  "mode_auto":{"val":1, "format":"0:Aus|1:Auto|2:Heizen|3:Absenken", "text":"Betriebsart"}, 
  "mode_dhw":{"val":0, "format":"0:Aus|1:Auto|2:Heizen|3:Absenken", "text":"Betriebsart"}, 
  "time_prg":{"val":0, "format":"0:Zeit 1|1:Zeit 2", "text":"Zeitauswahl"}, 
  "temp_setback":{"val":180, "unit":"°C", "factor":0.1, "min":100, "max":400, "text":"Raumtemp Absenken"}, 
  "temp_heat":{"val":220, "unit":"°C", "factor":0.1, "min":100, "max":400, "text":"Raumtemp Heizen"}, 
  "temp_vacation":{"val":150, "unit":"°C", "factor":0.1, "min":100, "max":400, "text":"Raumtemp Urlaub"}, 
  "name":{"val":"", "length":20, "text":"Anzeigename"}, 
  "oekomode":{"val":0, "format":"0:Aus|1:Komfort|2:Minimum|3:Ökologisch", "text":"Öko Modus"}
 },
 "pu1":{
  "pu_info":"accu data",
  "L_tpo_act":{"val":557, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Einschaltfühler"}, 
  "L_tpo_set":{"val":499, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"TPO Soll"}, 
  "L_tpm_act":{"val":411, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Abschaltfühler"}, 
  "L_tpm_set":{"val":499, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"TPM Soll"}, 
  "L_pump_release":{"val":557, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Pumpenfreigabe Temp"}, 
  "L_pump":{"val":100, "unit":"%", "factor":1, "min":0, "max":100, "text":"Drehzahl"}, 
  "L_state":{"val":256, "factor":1, "text":"Status"}, 
  "L_statetext":"Anforderung Ein",
  "mintemp_off":{"val":80, "unit":"°C", "factor":0.1, "min":80, "max":900, "text":"Puffertemp min Aus"}, 
  "mintemp_on":{"val":80, "unit":"°C", "factor":0.1, "min":80, "max":900, "text":"Puffertemp min Ein"}, 
  "ext_mintemp_off":{"val":80, "unit":"°C", "factor":0.1, "min":80, "max":900, "text":"Puffertemp min Aus"}, 
  "ext_mintemp_on":{"val":80, "unit":"°C", "factor":0.1, "min":80, "max":900, "text":"Puffertemp min Ein"}
 },
 "ww1":{
  "ww_info":"domestic hot water data",
  "L_temp_set":{"val":300, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Wassertemp Soll"}, 
  "L_ontemp_act":{"val":575, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Ein Temperatur"}, 
  "L_offtemp_act":{"val":557, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Aus Temperatur"}, 
  "L_pump":{"val":0, "format":"0:Aus|1:Ein", "text":"Pumpe"}, 
  "L_state":{"val":8200, "factor":1, "text":"Status"}, 
  "L_statetext":"Zeit außerhalb Zeitprogramm|Anforderung Aus",
  "time_prg":{"val":0, "format":"0:Zeit 1|1:Zeit 2", "text":"Zeitauswahl"}, 
  "sensor_on":{"val":0, "format":"0:WW|1:TPO|2:TPM|3:SpUnten", "text":"Einschaltfühler"}, 
  "sensor_off":{"val":1, "format":"0:WW|1:TPO|2:TPM|3:SpUnten", "text":"Abschaltfühler"}, 
  "mode_off":{"val":0, "format":"0:Aus|1:Auto|2:Ein", "text":"Betriebsart"}, 
  "mode_auto":{"val":1, "format":"0:Aus|1:Auto|2:Ein", "text":"Betriebsart"}, 
  "mode_dhw":{"val":1, "format":"0:Aus|1:Auto|2:Ein", "text":"Betriebsart"}, 
  "heat_once":{"val":0, "format":"0:Aus|1:Ein", "text":"Einmal Aufbereiten"}, 
  "temp_min_set":{"val":350, "unit":"°C", "factor":0.1, "min":80, "max":800, "text":"Wassertemp Min"}, 
  "temp_max_set":{"val":550, "unit":"°C", "factor":0.1, "min":80, "max":800, "text":"Wassertemp Soll"}, 
  "name":{"val":"", "length":20, "text":"Anzeigename"}, 
  "smartstart":{"val":0, "unit":"min", "factor":1, "min":0, "max":90, "text":"Intelligenter Start"}, 
  "use_boiler_heat":{"val":0, "format":"0:Aus|1:Ein", "text":"Restwärmenutzung"}, 
  "oekomode":{"val":0, "format":"0:Aus|1:Komfort|2:Minimum|3:Ökologisch", "text":"Öko Modus"}
 },
 "sk1":{
  "sk_info":"solar circuit data",
  "L_koll_temp":{"val":158, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Kollektortemp"}, 
  "L_spu":{"val":388, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Speicher Unten"}, 
  "L_pump":{"val":0, "unit":"%", "factor":1, "min":0, "max":100, "text":"Pumpe"}, 
  "L_state":{"val":32, "factor":1, "text":"Status"}, 
  "L_statetext":"Differenz Kollektor-Speicher zu niedrig",
  "mode":{"val":1, "format":"0:Aus|1:Ein", "text":"Betriebsart"}, 
  "cooling":{"val":0, "format":"0:Aus|1:Ökologisch|2:Ein", "text":"Solar kühlen"}, 
  "spu_max":{"val":850.0, "unit":"°C", "factor":0.1, "min":200, "max":900, "text":"Speichertemp. Max"}, 
  "name":{"val":"", "length":20, "text":"Anzeigename"}
 },
 "se1":{
  "se_info":"solar gain data",
  "L_flow_temp":{"val":374, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Vorlauftemperatur"}, 
  "L_ret_temp":{"val":315, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Rücklauftemperatur"}, 
  "L_flow":{"val":0, "unit":"l/min", "factor":0.01, "min":-32768, "max":32767, "text":"Durchfluss"}, 
  "L_pwr":{"val":0, "unit":"kW", "factor":0.1, "text":"Aktuell"}, 
  "L_counter":{"val":14070, "factor":1, "text":"Counter"}, 
  "L_total":{"val":950, "unit":"kWh", "factor":0.1, "min":0, "max":4.294967295E9, "text":"Gesamt"}, 
  "L_day":{"val":183, "unit":"kWh", "factor":0.1, "text":"Tagesertrag"}, 
  "L_yesterday":{"val":41, "unit":"kWh", "factor":0.1, "text":"Vortagesertrag"}
 },
 "circ1":{
  "circ_info":"circulation pump data",
  "L_pump":{"val":1, "format":"0:Aus|1:Ein", "text":"Pumpe"}, 
  "L_ret_temp":{"val":379, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Rücklauf"}, 
  "L_release_temp":{"val":575, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Freigabetemp"}, 
  "time_prg":{"val":0, "format":"0:Zeit 1|1:Zeit 2", "text":"Zeitauswahl"}, 
  "mode":{"val":1, "format":"0:Aus|1:Auto", "text":"Betriebsart"}, 
  "pump_release":{"val":300, "unit":"°C", "factor":0.1, "min":0, "max":900, "text":"Pumpenfreigabetemp"}, 
  "return_set":{"val":380, "unit":"°C", "factor":0.1, "min":0, "max":900, "text":"Abschalttemperatur"}, 
  "name":{"val":"", "length":20, "text":"Anzeigename"}
 },
 "pe1":{
  "pe_info":"pellematic data",
  "L_temp_act":{"val":579, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Kesseltemperatur"}, 
  "L_temp_set":{"val":700, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"KT Soll"}, 
  "L_ext_temp":{"val":-32768, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Abgastemperatur"}, 
  "L_frt_temp_act":{"val":6392, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Flammraumtemp"}, 
  "L_frt_temp_set":{"val":6677, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"FRT Soll"}, 
  "L_frt_temp_end":{"val":7500, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Flammraumendtemp"}, 
  "L_br":{"val":0, "format":"0:Aus|1:Ein", "text":"Brennerkontakt"}, 
  "L_ak":{"val":0, "format":"0:Aus|1:Ein", "text":"Best Kessel (AK)"}, 
  "L_not":{"val":1, "format":"0:Aus|1:Ein", "text":"Emergency Stop"}, 
  "L_stb":{"val":1, "format":"0:Aus|1:Ein", "text":"Safety Thermostat"}, 
  "L_modulation":{"val":100, "unit":"%", "factor":1, "min":-32768, "max":32767, "text":"Modulationsstufe"}, 
  "L_runtimeburner":{"val":3118, "unit":"zs", "factor":0.01, "text":"Einschubzeit"}, 
  "L_resttimeburner":{"val":8000, "unit":"zs", "factor":0.01, "text":"Pausenzeit"}, 
  "L_currentairflow":{"val":77, "unit":"%", "factor":1, "min":-32768, "max":32767, "text":"Lüfter Drehzahl"}, 
  "L_lowpressure":{"val":1461, "unit":"EH", "factor":0.1, "min":-32768, "max":32767, "text":"Unterdruck"}, 
  "L_lowpressure_set":{"val":1460, "unit":"EH", "factor":0.1, "min":-32768, "max":32767, "text":"Unterdruck Ende"}, 
  "L_fluegas":{"val":64, "unit":"%", "factor":1, "min":-32768, "max":32767, "text":"Saugzug Drehzahl"}, 
  "L_uw_speed":{"val":33, "unit":"%", "factor":1, "min":-32768, "max":32767, "text":"Drehzahl UW"}, 
  "L_state":{"val":16, "factor":1, "text":"PE Kesselstatus"}, 
  "L_statetext":"Leistungsbrand",
  "L_type":{"val":6, "format":"0:PE|1:PES|2:PEK|3:PESK|4:SMART V1|5:SMART V2|6:CONDENS|7:SMART XS|8:SMART V3|9:COMPACT|10:AIR|11:CONDENS XL", "text":"Kesseltyp"}, 
  "L_starts":{"val":3222, "factor":1, "text":"Brennerstarts"}, 
  "L_runtime":{"val":3165, "unit":"h", "factor":1, "text":"Brennerlaufzeit"}, 
  "L_avg_runtime":{"val":58, "unit":"min", "factor":1, "text":"PE Mittlere Laufzeit"}, 
  "L_uw_release":{"val":557, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Freigabetemperatur"}, 
  "L_uw":{"val":33, "unit":"%", "factor":1, "min":-32768, "max":32767, "text":"PE Drehzahl UW"}, 
  "L_storage_fill":{"val":2601, "unit":"kg", "factor":1, "text":"Füllstand Lager"}, 
  "L_storage_min":{"val":600, "unit":"kg", "factor":1, "min":0, "max":4000, "text":"Warnung bei"}, 
  "L_storage_max":{"val":6000, "unit":"kg", "factor":1, "min":150, "max":30000, "text":"Max. Füllmenge Lager"}, 
  "L_storage_popper":{"val":32, "unit":"kg", "factor":1, "min":-32768, "max":32767, "text":"Füllstand Zwischenb"}, 
  "storage_fill_today":{"val":14, "unit":"kg", "factor":1, "min":-32768, "max":32767, "text":"Pelletverbrauch heute"}, 
  "storage_fill_yesterday":{"val":26, "unit":"kg", "factor":1, "min":-32768, "max":32767, "text":"Pelletverbrauch gestern"}, 
  "mode":{"val":1, "format":"0:Aus|1:Auto|2:Ein", "text":"Betriebsart"}
 },
   "error":{
   }
}

**Comment by dominikamann** [comment]
Hi @trudetta,

please check & test the new release: https://github.com/dominikamann/oekofen-pellematic-compact/releases/tag/v3.1.2

All the best,
Dominik

**Reply by trudetta** [reply]
Great, the graph view is now working. But I guess you miss to multiply by the factor ("factor":0.1). Can you change this? Thanks a lot in advance.

![IMG_4008](https://github.com/dominikamann/oekofen-pellematic-compact/assets/155584372/0765662d-000e-4b12-b5cf-b1144f46db36)
![IMG_4007](https://github.com/dominikamann/oekofen-pellematic-compact/assets/155584372/b58790c3-0c4b-4005-a626-43c4cf97901f)

**Reply by dominikamann** [reply]
@trudetta can you please provide the json with the current values again and a picture of the value total, yesterday and today. Section "SE1" of the json is enough and please only the values for today. Thanks.

**Reply by trudetta** [reply]
"L_day":{"val":0, "unit":"kWh", "factor":0.1, "text":"Tagesertrag"}, 
  "L_yesterday":{"val":99, "unit":"kWh", "factor":0.1, "text":"Vortagesertrag"}

**Reply by dominikamann** [reply]
And in Home Assistant it is 99 kWh for Yesterday?

**Comment by trudetta** [comment]
Yes, right

sent from my mobile device
________________________________
Von: Dominik Amann ***@***.***>
Gesendet: Monday, March 11, 2024 9:32:26 PM
An: dominikamann/oekofen-pellematic-compact ***@***.***>
Cc: trudetta ***@***.***>; Mention ***@***.***>
Betreff: Re: [dominikamann/oekofen-pellematic-compact] Data display of „Solar Gain Today“ different to „Pellet Usage Today“ (Discussion #45)


And in Home Assistant it is 99 kWh for Yesterday?

**Comment by dominikamann** [comment]
Ok that's strange because my integration takes the value and divides it with 10.  So for "Yesterday" it should be 9,9 kw/h. Iam not able to test it as i dont use this functionality with my heater.

**Reply by trudetta** [reply]
can you try instead to multiply by 0,1? Actually should be the same but maybe it helps.

here a screenshot of the statistic, usage of pellets and solar gain (both today)
![image](https://github.com/dominikamann/oekofen-pellematic-compact/assets/155584372/f64d1781-4244-4071-b22b-daab8227d8df)

Strange is that the dasboard statistic chart is not working for the solar gain but it is working for the usage, do you have an explanation for this?
![image](https://github.com/dominikamann/oekofen-pellematic-compact/assets/155584372/627a1ee8-3028-4cb9-905b-0f7332d6ef1e)

**Comment by dominikamann** [comment]
@trudetta thank you for the screenshots i found the issue. I released a new version. Please install & test.

**Reply by trudetta** [reply]
für today klappt es noch nicht,.. vielleicht ändert sich die Umrechnung auch nur für neu gespeicherte Werte?! bei total sieht man einen Sprung (Faktor 10 Kleiner) nach Installation der neuen Version.

![image](https://github.com/dominikamann/oekofen-pellematic-compact/assets/155584372/fdcec41b-1e4f-4e5e-9691-fc3fa02e6506)

![image](https://github.com/dominikamann/oekofen-pellematic-compact/assets/155584372/9feb5f91-d9c1-439d-9778-90a22697fdfe)

**Reply by dominikamann** [reply]
Das wird voraussichtlich nur für neue Werte angewandt. Bitte beobachten.

**Comment by dominikamann** [comment]
Was genau ist das Problem mit dem Statistikdiagramm? Werden keine Werte angezeigt? Es ist wichtig zu wissen, dass ich erst mit dem gestrigen Update auf die Zustandsklasse "Total_Increasing" für die 3 Sensoren (Total, Yesterday, Today) umgestellt habe. Das sollte dann mit der HA-Statistik funktionieren. Auch im Energie-Dashboard. Siehe https://developers.home-assistant.io/blog/2021/08/16/state_class_total/. Vielleicht hat es damit zu tun, dass die Erstellung der Statistiken noch etwas Zeit benötigt oder noch zu wenig Werte da sind. Die letzen 7 Tage werden vermutlich nicht funktionieren.
